client/src/views/owner-home.js

```diff
@@ -0,0 +1,107 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { Title } from "../components/ui";
+import OwnerSportFieldCard from "../components/owner-sportfield-card";
+import OwnerClubCircle from "../components/owner-club-circle";
+import OwnerCreateButton from "../components/owner-create-button";
+import { useSetClubs, useSetSportFields } from "../actions/action-hooks";
+import { useClubs, useSportFields } from "../selectors/selectors";
+import { getClubs } from "../services/club";
+import { getSportFields } from "../services/sportField";
+
+function OwnerHome() {
+  const clubs = useClubs();
+  const sportFields = useSportFields();
+  const setClubs = useSetClubs();
+  const setSportFields = useSetSportFields();
+  const [activeClub, setActiveClub] = React.useState(1);
+
+  const styleClubsContainer = {
+    display: "flex",
+    alignItems: "center",
+    overflowX: "auto",
+    padding: "1em 0",
+    "&::-webkit-scrollbar-track": {
+      boxShadow: "inset 0 0 6px rgba(0,0,0,0.3)",
+      borderRadius: "10px",
+      backgroundColor: "#F5F5F5"
+    },
+    "&::-webkit-scrollbar": {
+      height: "10px",
+      backgroundColor: "#F5F5F5"
+    },
+    "&::-webkit-scrollbar-thumb": {
+      borderRadius: "10px",
+      boxShadow: "inset 0 0 6px rgba(0,0,0,.3)",
+      backgroundColor: "#8f8f8f"
+    }
+  };
+
+  React.useEffect(() => {
+    getClubs().then(clubs => {
+      setClubs(clubs);
```

Comment:
You could pass the default value of the club(`activeClub`) here:
```
React.useEffect(() => {
    getClubs().then(clubs => {
      setClubs(clubs);
      setActiveClub(clubs.length ? clubs[0].id : null)
   });
 }, [setClubs]);
```

Reply:
Thanks for noticing all of that, I'll change it right now.